gnomad/utils/annotations.py

```diff
@@ -1247,81 +1251,84 @@ def get_gks(
         & (ht.alleles == [ref_in, alt_in])
     )
 
```

Comment:
```suggestion
ht = ht.checkpoint(
        new_temp_file("get_gks", extension="ht"))
```

Reply:
add "from hail.utils.misc import new_temp_file"

checkpointing here should make things faster

Reply:
added to the top of the script, will have to run isort and reformat in my next commit once I pull the changes and run isort